pytorch_lightning/trainer/trainer.py

```diff
@@ -2026,10 +2036,18 @@ def tpu_cores(self) -> int:
     def ipus(self) -> int:
         return self._accelerator_connector.num_ipus
 
+    @property
+    def hpus(self) -> int:
+        return self._accelerator_connector.num_hpus
+
     @property
     def num_gpus(self) -> int:
         return self._accelerator_connector.num_gpus
 
+    @property
+    def num_hpus(self) -> int:
+        return self._accelerator_connector.num_hpus
```

Comment:
same comment as above

Reply:
Will it be removed for all backends ?

Reply:
Am keeping it for now. Let me know if this really needs to be removed

Reply:
This will can replaced with num_devices(), same as num_gpus
This PR will be merge as soon as CI unblock https://github.com/PyTorchLightning/pytorch-lightning/pull/12151 , this will unblock num_x properties. 
Feel free to update here after the PR above merged

Reply:
updated